src/sas/sascalc/dataloader/data_info.py

```diff
@@ -92,14 +39,17 @@ class plottable_1D(object):
     dxl = None
     ## Slit smearing width
     dxw = None
+    ## SESANS specific params (wavelengths for spin echo length calculation)
+    lam = None
+    dlam = None

```

Comment:
We need to rethink our handling of data; the current code is far to complex with copies being made in several places with slightly different attributes and lots of special code asking if it is 1D data, 2D data and now SESANS data.  We will also need something similar for oriented USANS data so that the model can include orientation parameters even though it is a 1D dataset, with the resolution calculator returning a 1D data set from the 2D calculation.

My preference would be for you to add the lam/dlam attributes when you need them so that refactoring is less work.  Also, I suspect you won't need them everywhere that you've added them, but we won't know that until we have the code that uses them.